### ISSUE TITLE
Is there a way to restore a custom session from the command line

### ISSUE THREAD

**Issue by AlfonsoRReyes** [nlpsuge/gnome-shell-extension-another-window-session-manager#91]
Hello,
I have an automated script to update packages, extensions, and snaps.
For the snaps, specifically for Firefox, I have to kill it before updating (refresh) the snap.
The problem is restarting firefox with a simple `firefox &`, restores a different  set of windows disregarding the workspaces set by AWSM. Obvious.

My question is if there is way to restore an AWSM named session, let's say, MySession, from the command line?

Thanks.

**Comment by p0nti** [comment]
I agree, that way one could assign a keyboard shortcut and make this whole process even more streamlined.

**Comment by Francewhoa** [comment]
Maybe this workaround at https://github.com/nlpsuge/gnome-shell-extension-another-window-session-manager?tab=readme-ov-file#support-applications-launched-via-a-command-line-or-applications-that-dont-have-a-proper-desktop-file
